.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
 #  - sudo apt-cache madison docker-engine
 
   # update docker engine to the desired version
-  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-ce=17.07.0~ce-0~ubuntu
+  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-ce=17.09.0~ce-0~ubuntu
```

Comment:
I'm confused. Why is this even a diff? This is what I did in my commit: https://github.com/ruflin/Elastica/blob/master/.travis.yml#L28

You had a nicer version in your previous PR where it was a variable.

Reply:
yes with the ENV variables